lib/changelog_web/templates/feed/topic.xml.eex

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
+  <channel>
+    <title>Changelog: <%= @topic_name %></title>
+    <copyright>All rights reserved</copyright>
+    <link><%= Routes.root_url(@conn, :index) %></link>
+    <atom:link href="<%= Routes.feed_url(@conn, :news) %>" rel="self" type="application/rss+xml" />
+    <atom:link href="<%= Routes.root_url(@conn, :index) %>" rel="alternate" type="text/html" />
+    <language>en-us</language>
+    <description><%= @topic_description %></description>
+  <%= for item <- @items do %>
+    <item>
+      <%= render_item(item, assigns) %>
```

Comment:
This pattern matches to `  def render_item(item = %{object: nil}, assigns) do ...` because a `NewsItem` struct is returned. Maybe the query should be changed so that it returns only the preloaded posts of the specific topic. wdyt? :thinking: